app/src/main/java/org/listenbrainz/android/service/ApiService.kt

```diff
@@ -0,0 +1,148 @@
+package org.listenbrainz.android.service
+
+import org.listenbrainz.android.model.CoverArt
+import org.listenbrainz.android.model.ListenBrainzExternalServices
+import org.listenbrainz.android.model.ListenSubmitBody
+import org.listenbrainz.android.model.Listens
+import org.listenbrainz.android.model.PinData
+import org.listenbrainz.android.model.PinnedRecording
+import org.listenbrainz.android.model.PostResponse
+import org.listenbrainz.android.model.RecommendationData
+import org.listenbrainz.android.model.Review
+import org.listenbrainz.android.model.SearchResult
+import org.listenbrainz.android.model.SimilarUserData
+import org.listenbrainz.android.model.SocialData
+import org.listenbrainz.android.model.SocialResponse
+import org.listenbrainz.android.model.TokenValidation
+import org.listenbrainz.android.model.feed.FeedData
+import org.listenbrainz.android.model.feed.FeedEvent
+import org.listenbrainz.android.model.feed.FeedEventDeletionData
+import org.listenbrainz.android.model.feed.FeedEventVisibilityData
+import org.listenbrainz.android.util.Constants.Headers.AUTHORIZATION
+import retrofit2.Response
+import retrofit2.http.Body
+import retrofit2.http.GET
+import retrofit2.http.Header
+import retrofit2.http.POST
+import retrofit2.http.Path
+import retrofit2.http.Query
+
+interface ApiService {
```

Comment:
@07jasjeet can you remove this file? Also, just ping me when to review this PR.

Reply:
Mb forgot this one

Reply:
Ready to review @akshaaatt